third_party/requirements.txt

```diff
@@ -6,7 +6,7 @@ black==24.3.0
 # Tests
 packaging==24.2
 # Generating HTML documentation
-pygments==2.19.1
+pygments==2.18.0
```

Comment:
you should not change it back to prev version (probably changed with conflict resolution)